lib/anno_utils.pm

```diff
@@ -0,0 +1,2917 @@
+# -*- perl -*-
+
+package anno_utils;
```

Comment:
Perl packages should be named with capital letters, and write out the whole word -- `package AnnotationUtils;`.